### DISCUSSION TITLE
希望大佬支持链式代理，nekobox的链式代理功能用起来太舒服了

### DISCUSSION THREAD

**Discussion by liyiranlab** [immortalwrt/homeproxy#88] [Ideas]
希望大佬支持链式代理，nekobox的链式代理功能用起来太舒服了，用WireGuard落地，垃圾机场节点也跑满100m。试过 openclash 用不了WireGuard。cfw可以使用 dialer-proxy 实现，希望直接在路由器上跑起来。

**Comment by mqcycy** [comment]
理解一下插件的名字，一个自建节点开箱即用的插件，没必要搞那复杂

**Comment by nellwhoami** [comment]
> 希望大佬支持链式代理，nekobox的链式代理功能用起来太舒服了，用WireGuard落地，垃圾机场节点也跑满100m。试过 openclash 用不了WireGuard。cfw可以使用 dialer-proxy 实现，希望直接在路由器上跑起来。

可以再docker 一个3xui ,打组合拳
